src/Security/Security/Cmdlets/SqlVulnerabilityAssessment/SetSqlVulnerabilityAssessmentBaseline.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Commands.Security;
+using Microsoft.Azure.Commands.Security.Common;
+using Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment;
+using Microsoft.Azure.Management.Security.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Cmdlets.SqlVulnerabilityAssessment
+{
+    [Cmdlet(VerbsCommon.Set, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlVulnerabilityAssessmentBaseline", SupportsShouldProcess = true), OutputType(typeof(PSSqlVulnerabilityAssessmentBaselineResults))]
```

Comment:
Please define a default parameter set in the cmdlet attribute.

Reply:
pending